types/osmosis/index.d.ts

```diff
@@ -54,7 +55,9 @@ export interface Osmosis {
      * passing string to your function
      * result data, osmosis finished
      */
-    data(callback: (param: string) => any): Osmosis;
-}
+    data(callback: (param: any) => any): Osmosis;
```

Comment:
why is this `any`?

Reply:
Typical use will probably be something along the lines of this:

```ts
interface CollectedItem {
  title: string
}

osmosis
    .get('https://lists.acme.com')
    .find('.list-item')
    .set('title')
    .data((data: CollectedItem) => {})
```

The eventual shape of the data is determined by the function chain so it's not possible to have a specific type for it AFAICT.

Reply:
true, thx for the fix